src/App.tsx

```diff
@@ -549,16 +549,16 @@ export default function App() {
                   <InventoryCreate organization={selectedOrganization} />
                 </Route>
               )}
-              {selectedOrganization && (
-                <Route path={APP_PATHS.INVENTORY_ITEM}>
-                  <InventoryView organization={selectedOrganization} species={species} />
-                </Route>
-              )}
               {trackingEnabled && selectedOrganization && (
                 <Route path={APP_PATHS.INVENTORY_WITHDRAW}>
                   <SpeciesBulkWithdrawWrapperComponent organization={selectedOrganization} />
                 </Route>
               )}
+              {selectedOrganization && (
```

Comment:
moved this down so /inventory/withdraw and /inventory/:speciesId have different routing